uninstall.sh

```diff
@@ -0,0 +1,326 @@
+#!/bin/bash
+
+set -e
```

Comment:
We discussed this on Discord. What are the pros and cons of using `set -e` for the uninstallation script? I can think of a few myself but I am open for dicussion.